File: create_dir.py
import os

directory_path = r"c:\Cosas\Rol\rol\src\test\java\com\dungeons_and_dragons\rol\integration"

try:
    os.makedirs(directory_path, exist_ok=True)
    if os.path.isdir(directory_path):
        print("✓ Directory created successfully!")
        print(f"Path: {directory_path}")
        print(f"Exists: {os.path.exists(directory_path)}")
    else:
        print("✗ Failed to create directory")
except Exception as e:
    print(f"✗ Error: {e}")

File: create_dir.js
const fs = require('fs');
const path = require('path');

const dirPath = String.raw`c:\Cosas\Rol\rol\src\test\java\com\dungeons_and_dragons\rol\integration`;

try {
    fs.mkdirSync(dirPath, { recursive: true });
    if (fs.existsSync(dirPath)) {
        console.log('✓ Directory created successfully!');
        console.log(`Path: ${dirPath}`);
        console.log(`Exists: ${fs.existsSync(dirPath)}`);
    } else {
        console.log('✗ Failed to create directory');
    }
} catch (error) {
    console.log(`✗ Error: ${error.message}`);
}
